Sources/SkipBluetooth/CBATTRequest.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
```

Comment:
Can you add the LGPL license header to this and the other files so they all have the same header?

Reply:
Sure thing, completed.